data/json/items/armor/helmets.json

```diff
@@ -499,13 +499,41 @@
     "material": [ "plastic", "nomex" ],
     "symbol": "[",
     "color": "dark_gray",
-    "armor": [
-      { "covers": [ "head" ], "coverage": 100, "encumbrance": 25 },
-      { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 5 },
-      { "covers": [ "mouth" ], "coverage": 100, "encumbrance": 15 }
-    ],
+    "armor_data": {
+      "material_thickness": 6,
+      "armor": [
+        { "covers": [ "head" ], "coverage": 100, "encumbrance": 25 },
+        { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 5 },
```

Comment:
I don't know if it's possible to set an material thickness specifically for one section,  I'd think this would have 1 or 2 thickness when visor down over the eyes.  But again not sure that's how it works.

Reply:
You cannot.